Code/wwphys/Path.cpp

```diff
@@ -1098,7 +1098,11 @@ PathClass::Initialize_Vehicle_Spline (DynamicVectorClass<PATH_NODE> &node_list)
 		//	Add this point as a key along the spline
 		//
 		current_dist		+= (point - last_point).Length ();
-		float curr_time	= current_dist / m_TotalDist;
+
+		float curr_time = 0.0f;
+		if (m_TotalDist != 0.0f) {
+			curr_time = current_dist / m_TotalDist;
```

Comment:
I don't think that this is really in scope for this change, but comparing to `0.0f` is almost always going to be false.  It really needs to be comparing with an epsilon value.

Reply:
I also think it's fine. An epsilon would keep an object on a small spline path on the start point.